Docs for MessageConfigurator (#528)
vaadin/docs: articles/ce/collaboration-message-list.asciidoc

through the provided `ActivationContext`.
The callback should return another callback for clean-up when the topic connection is deactivated (topic id is changed).
Note how the submit button is enabled only when there's an active topic connection.

== Customizing Messages
`CollaborationMessageList` enables changing the properties of `MessageListItem` after they are generated.
This is done by providing a `MessageConfigurator`.

> Here is an example of how to use a message configurator to add a custom style to the current user's messages.
> Refer to <<{articles}/ds/components/messages#theming, messages theming documentation>> for details on how style messages based on the theme.
> [source,java]
> ----
> include::{root}/src/main/java/com/vaadin/demo/ce/MessageListDocumentation.java[tags=message-list-configurator;message-list-configurator-style;!*,indent=0]
> ----

Reviewer: ```suggestion
The following example shows how to use a message configurator to add a custom style to the current user's messages:

[source,java]
----
include::{root}/src/main/java/com/vaadin/demo/ce/MessageListDocumentation.java[tags=message-list-configurator;message-list-configurator-style;!*,indent=0]
----

Please refer to <<{articles}/ds/components/messages#theming, Theming>> in Messages documentation for details on how to style messages based on the theme.
```

> The message configurator can also be used to change the message's text, for example censoring certain words in messages.

Reviewer: ```suggestion
The message configurator can also be used to change the text content of messages, for example, to censor certain words in messages, as is done in the following example.
```
